README.md

```diff
@@ -1 +1,22 @@
-# To-do-testing
\ No newline at end of file
+## To-do-testing
+![](https://img.shields.io/badge/Microverse-blueviolet)
+## Description
+
+This is the first testing project with jest and simulated the html with jest-environment-jsdom to testing some many instance of the code.
+
+## Built with :
+
+- javaScript
+- jest 
+
```

Comment:
- [ ] Please give detailed instructions on how users can set up your project on their local machines 👍 
- [ ] Also, it would be really great if you could tell users how to run the tests